app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,24 @@
+<nav class="navbar is-fixed-top" role="navigation" aria-label="main navigation">
+  <div id="navbarBasicExample" class="navbar-menu">
+    <div class="navbar-start">
+      <% if user_signed_in? %>
+        <%= link_to "Food", foods_path, class: "navbar-item has-text-link" %>
+        <%= link_to "Recipes", recipes_path, class: "navbar-item has-text-link" %>
+        <%= link_to "Public recipes", recipes_path, class: "navbar-item has-text-link" %>
+        <%= link_to "Shopping list", recipes_path, class: "navbar-item has-text-link" %>
+      <% else %>
+        <%= link_to "Sign up", new_user_registration_path %> |
+        <%= link_to "Login", new_user_session_path %>
+      <% end %>
+    </div>
```

Comment:
- We don't need to add any logic inside the .erb file, we use the helper folders to create methods with logic and called inside the .erb file. You can refer to this comment to modify the files that have logic inside the .erb file.

Read [this](https://www.toptal.com/ruby-on-rails/top-10-mistakes-that-rails-programmers-make) for more information